app/views/jobs/new.html.erb

```diff
@@ -1 +1,13 @@
-new.html.erb
+<div class="container">
+  <h1>Create a Job</h1>
+
+  <%= simple_form_for [ @job ] do |f| %>
```

Comment:
I think we need to pass `current_user` to the new isntance of job. We can either do it here or in the controller. I think it might be easier to do in the controller becuase then we don't need to deal with hidden fields etc. We also need to add a `category_id` field becuase the job model `belongs_to` category so without it it won't be created.